### PR TITLE
removing reading of var log messages*

### PR DESCRIPTION
**Pull request by ewolinetz**

### CODE REVIEW

**Comment by ewolinetz** [comment]
Should be all that there is to it...  [test] to make sure the filter update doesn't blow up?

**Comment by richm** [comment]
What about the logic in the fluentd config that attempts to determine what the year is for a timestamp from /var/log/messages that appears to be in the future?

**Comment by ewolinetz** [comment]
@richm that _should_ just be in `filter-syslog-record-transform.conf` ...

**Comment by richm** [comment]
@ewolinetz right, but does that logic need to change at all?

**Comment by ewolinetz** [comment]
@richm ah, yes since the tag parts go from `["system","var","log","messages-20150710"]` to `["system","var","log","messages"]` there isn't a reason to maintain that ugly line of code.  It is also to address https://bugzilla.redhat.com/show_bug.cgi?id=1347871

**Comment by ewolinetz** [comment]
ah, the "currently deployed" issue... lets just [test] and i'll try to fix it later, eh?

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
etcd.. you're killing me [test]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
Didn't see anything fail/any error messages in fluentd/es.. but also not seeing new logs being populated in ES... might try to add some waits for logs to populate?

Also didn't see the logic for an existing deploy get used with latest test...

**Comment by ewolinetz** [comment]
Let's [test] again

File: hack/testing/logging.sh [hunk 1]
```diff
@@ -488,6 +488,9 @@ function reinstall() {
   os::cmd::try_until_text "oc get pods -l component=kibana" "Running" "$(( 3 * TIME_MIN ))"
   os::cmd::try_until_text "oc get pods -l component=curator" "Running" "$(( 3 * TIME_MIN ))"
   os::cmd::try_until_text "oc get pods -l component=fluentd" "Running" "$(( 3 * TIME_MIN ))"
+
+  # wait 3 minutes to let Fluentd process some records to send to ES
+  sleep 180
```
Comment (richm) [inline hack/testing/logging.sh]: This seems excessive - if it is really taking this long, we need to find out why

Comment (ewolinetz) [inline hack/testing/logging.sh]: This probably is more than enough time... It's more of a test to see if adding time allows for records to flow into ES for the upgrade/reinstall tests

**Comment by ewolinetz** [comment]
Only saw an improvement with the reinstall test...

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test] so close...

**Comment by ewolinetz** [comment]
@richm @sosiouxme PTAL

**Comment by ewolinetz** [comment]
lets [test] again to verify the `oc deploy` hiccup

**Comment by ewolinetz** [comment]
that last run captured the deploy issue we kept encountering... success!

**Comment by sosiouxme** [comment]
LGTM

**Comment by richm** [comment]
LGTM

**Comment by ewolinetz** [comment]
[merge]

**Comment by ewolinetz** [comment]
etcd issue...  [merge]

**Comment by ewolinetz** [comment]
c'mon, do it nao! [merge]

**Pull request merged**